jinahub/indexers/dbms/LMDBDBMSIndexer/README.md

```diff
@@ -0,0 +1,6 @@
+# LMDBDBMSIndexer
```

Comment:
Please add clear description of all parameters

Reply:
added

Reply:
This name looks very clumsy. We need to find a better naming scheme for `DBMSIndexer`. I even wonder if we should rename to `FooIndexer` and `BarSearcher` and skip the DBMS completely. Somehow I remember we once had a discussion about it, but can't remember the arguments anymore. Anyhow, with 2.0 it might be the right point in time to do this change.